wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/ParallelCommandGroup.java

```diff
@@ -19,7 +19,7 @@
  */
 public class ParallelCommandGroup extends Command {
   // maps commands in this composition to whether they are still running
-  private final Map<Command, Boolean> m_commands = new HashMap<>();
+  private final Map<Command, Boolean> m_commands = new LinkedHashMap<>();
```

Comment:
A comment explaining that LinkedHashMap is used to define element order would be helpful. (Applies for the other places as well).